### ISSUE TITLE
Coq not completing imported libraries

### ISSUE THREAD

**Issue by zbirenbaum** [ms-jpq/coq_nvim#355]
So I was doing some updating but I think something went wrong, and now while my LSP server and Coq both are starting up properly (I have confirmed both) and coq claims to show LSP completions, if I say, import pandas in a python file, no completions for pandas methods or classes will appear in coq. I have tried to do the 'ensure capabilities' about a million different ways but nothing seems to work. Really hoping I can get some help here.

**Comment by zeertzjq** [comment]
Does completion appear if you press `<C-Space>`?

**Comment by zbirenbaum** [comment]
> Does completion appear if you press `<C-Space>`?

Yes actually that did work. Why does that cause imported libraries to appear? Is there a way to make it the default?

edit: now that is not working, I'm very confused

**Comment by zeertzjq** [comment]
<https://github.com/ms-jpq/coq_nvim#lsp-sometimes-not-importing>

**Comment by zbirenbaum** [comment]
> https://github.com/ms-jpq/coq_nvim#lsp-sometimes-not-importing

Ah thank you, but I am still unsure why it could be that the manual completion mode would change the actual completion options compared to the normal menu. In addition, not even that works now.

**Comment by zbirenbaum** [comment]
Ok so I figured it out I think. Turns out one of the lines in my pyright config was causing issues for some reason and analysis was taking ages. The reason why C-space worked isn't because it did anything, but that I did it a sufficient amount of time after opening the buffer for analysis to complete. It now works, but performance still seems lower than before, however that's the fault of pyright not coq.
